ret-core/src/main/kotlin/io/rabobank/ret/util/RegexUtils.kt

```diff
@@ -1,5 +1,11 @@
 package io.rabobank.ret.util
 
+/**
+ * Regex Utilities
```

Comment:
I would move the doc for the pattern to the pattern. The object level doc has no additional value.